### ISSUE TITLE
Pie Chart "Agents" in Tab Overview/OpenSCAP does not show all Agents

### ISSUE THREAD

**Issue by juergenc** [wazuh/wazuh-kibana-app#918]
### Release Level
all components: _3.6.1_

### Situation
* ~22 agents are configured for CVE vulnerability test via the OVAL files
* mixed OS, Debian 8 and 9 (shouldn't be relevant here)
* in _Kibana_, only _5 of 22_ agents are shown in the first pie chart _Agents_ in the first chart column of the _Overview/OpenSCAP_ tab
* results for all other agents with enabled OSCAP-stack are definitely stored in Elastic Search, you can find the alerts with the _Discovery_ in _Kibana_

**Comment by JuanjiJG** [comment]
Hello @juergenc,

That's correct. The first four pie chart visualizations on the _Overview > OpenSCAP_ tab are designed to only show a top 5 of agents.
![image](https://user-images.githubusercontent.com/10928321/46192652-14202c00-c2fc-11e8-8992-b9c8462d248c.png)

We'll fix this by changing the title so the users don't get confused. Apologies for the inconveniences.

As you can see here (code lines):
- [Agents](https://github.com/wazuh/wazuh-kibana-app/blob/c6d6425fc26d38cd5d86859bda23fff3a6d1a95a/server/integration-files/visualizations/overview/overview-oscap.js#L99)
- [Profiles](https://github.com/wazuh/wazuh-kibana-app/blob/c6d6425fc26d38cd5d86859bda23fff3a6d1a95a/server/integration-files/visualizations/overview/overview-oscap.js#L145)
- [Content](https://github.com/wazuh/wazuh-kibana-app/blob/c6d6425fc26d38cd5d86859bda23fff3a6d1a95a/server/integration-files/visualizations/overview/overview-oscap.js#L191)
- [Severity](https://github.com/wazuh/wazuh-kibana-app/blob/c6d6425fc26d38cd5d86859bda23fff3a6d1a95a/server/integration-files/visualizations/overview/overview-oscap.js#L237)

You can find something like this:
```
{"orderBy": "1", "field": "agent.name", "order": "desc", "size": 5}
```

The `size: 5` property indicates the number of agents to show by its name. If you want to try, you can also modify it editing the proper file, located at:
```
/usr/share/kibana/plugins/wazuh/server/integration-files/visualizations/overview/overview-oscap.js
```

And then, restart the Kibana service:
```
systemctl restart kibana
```

[This Elastic guide](https://www.elastic.co/guide/en/kibana/current/visualize.html) can give you more information about how visualizations are made.

Let us know if you have more questions or problems.

Regards,
Juanjo

**Comment by juergenc** [comment]
Hello @JuanjiJG ,

indeed, the missing _Top 5_ puzzled me. Alright, I understood.
I always appreciate you tips and hints, willingly to help giving back more than expected and letting me "look behind the curtains" :)
As I can see you have the change of the title on your list, so I close this ticket.

Muchísmas gracias,
Jorge

**Comment by JuanjiJG** [comment]
Happy to help you @juergenc, don't hesitate to open a new issue if you have more problems or questions about Wazuh.

We'll get that fix (#923) ready for the next package.

Regards,
Juanjo
